kolibri/core/assets/src/core_app.js

```diff
@@ -43,4 +43,4 @@ var Kolibri = function() {
     _.extend(this, _.pick(mediator, public_methods));
 };
 
-module.exports = Kolibri;
+module.exports = new Kolibri();
```

Comment:
Not a big fan of this de-abstraction, as it makes unit testing the facade itself more difficult, as we never directly export the Kolibri constructor.